pyoozie/coordinator.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+from __future__ import unicode_literals
+
+from datetime import timedelta
+from enum import Enum
+
+from pyoozie.tags import _validate, Xml, Parameters, Configuration
+
+
+class ExecutionOrder(Enum):
+    """Execution order used for coordinator jobs."""
+
+    FIFO = 'FIFO'
+    LIFO = 'LIFO'
+    LAST_ONLY = 'LAST_ONLY'
+    NONE = 'NONE'
+
+    def __str__(self):
+        return self.value
+
+
+def format_datetime(value):
+    return value.strftime('%Y-%m-%dT%H:%MZ')
```

Comment:
Oozie's time grain is `minute`. You can probably specify seconds, but they'd be ignored.
From https://oozie.apache.org/docs/4.1.0/CoordinatorFunctionalSpec.html#a4._Datetime_Frequency_and_Time-Period_Representation

> If the Oozie processing timezone is UTC , all datetime values are always in UTC down to a minute precision, 'YYYY-MM-DDTHH:mmZ'.